android/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.amazonaws.ivs.reactnative.player">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"  >
```

Comment:
What does dropping the `package="com.amazonaws.ivs.reactnative.player"` prop here do?
Is it related to moving the namespace info android/build.gradle ?

Reply:
It's part of the [changes](https://github.com/react-native-community/discussions-and-proposals/issues/671) necessary to make this lib compatible with RN 0.73

Reply:
I believe other changes, such as the Gradle plugin can be made after RN 0.73 is released, but this small change will at least bring minimum compatibility.

Reply:
awesome, thank you 👍